src/renderer/library/components/Wizard.tsx

```diff
@@ -100,8 +103,27 @@ export const WizardModal = () => {
                             <h4>{__("wizard.tab.annotations")}</h4>
                         </Tabs.Trigger>
                         <div style={{display: "flex", alignItems: "center", gap: "10px", position: "absolute", bottom: "30px", left: "30px"}}>
-                            <input type="checkbox" checked={checked} onChange={() => setChecked(!checked)} id="wizardCheckbox" name="wizardCheckbox" />
-                            <label htmlFor="wizardCheckbox">{__("wizard.dontShow")}</label>
+                            <input type="checkbox" checked={checked} onChange={() => setChecked(!checked)} id="wizardCheckbox" name="wizardCheckbox" className={stylesGlobal.checkbox_custom_input} />
+                            <label htmlFor="wizardCheckbox" className={stylesGlobal.checkbox_custom_label}>
+                                <div
+                                    tabIndex={0}
+                                    role="checkbox"
+                                    onKeyDown={(e) => {
+                                        if (e.key === " ") {
```

Comment:
"Space" please (not " ") for consistency with existing code

Reply:
@danielweck I see something strange on your PR the "Space" is on the `key` value but on my test that works only with the `code` value not `key` ("Space" not referenced in https://www.w3.org/TR/uievents-key/#named-key-attribute-values ) 
And if I follow your PR (https://github.com/edrlab/thorium-reader/pull/2269/files) you replace `" "` on `key` value. 

Is it working on keyPress and not on keyDown/keyUp ?

Reply:
see control character : https://www.w3.org/TR/uievents-key/#control